aviary/docs/examples/reserve_missions.ipynb

```diff
@@ -44,7 +44,7 @@
     "expected_values = {\n",
     "    'reserve': True,\n",
     "    'target_distance': (200, 'km'),\n",
```

Comment:
considering we have 'distance_solve_segments' wouldn't it make sense to update 'target_distance' to 'distance_target' ?

Reply:
This is actually an oddball one because the state is called `distance`, but target_distance is really more like target_delta_distance. If we wanted to generalize this to other states like mass, `mass_target` wouldn't imply end_mass - start_mass.  I think we could come up with better name though.